tests/pouchdb-integration/tests.js

```diff
@@ -92,7 +92,7 @@ function destroyObjects (done) {
   }, done)
 }
 
-describe('Replication stack -> pouchdb', function () {
+xdescribe('Replication stack -> pouchdb', function () {
```

Comment:
Was it just for tests and forgotten? Or is there a good reason to disable those tests?